tests/library/inspector/cli-codegen-3.spec.ts

```diff
@@ -116,6 +124,14 @@ await page.GetByRole(AriaRole.Button, new() { Name = "Submit" }).Nth(1).ClickAsy
 
     expect.soft(sources.get('C#')!.text).toContain(`
 await page.Locator("#frame1").ContentFrame.GetByText("Hello1").ClickAsync();`);
+
+    const clickAction = sources.get('JSON')!.actions.map(l => JSON.parse(l)).find(a => a.name === 'click');
+    expect.soft(clickAction).toMatchObject({
```

Comment:
Since you are serializing internal type FrameDescription, make sure it does not leak newly added params in test - replace `toMatchObject` with `toEqual`.